test/conftest.py

```diff
@@ -166,15 +166,15 @@ def mocked_chat_postMessage(self, **kwargs):
         Mocks chat.postMessage api call.
         '''
         channel_id_or_name = kwargs.get('channel')
-        text = kwargs.get('text')
 
         channel = self.channels.get(channel_id_or_name)
         if channel is None:
             return {'ok': False}
 
-        message = {'text': text, 'ts': str(time.time())}
+        message = {'ts': str(time.time()), **kwargs}
         self.test_messages[channel.id].append(message)
-        message_event = {'type': 'message', 'channel': channel.id, **message}
+        message_event = {'type': 'message', **message}
+        message_event['channel'] = channel.id
```

Comment:
why is this brought outside?

Reply:
The message event is what the bot would receive, which I'm pretty sure is always a channel id. The splat was overwriting it with whatever the arg passed in was.